src/services/FileService.ts

```diff
@@ -19,17 +20,45 @@ class FileService {
   }
 
   /**
-   * Get presigned URL for file upload
+   * Get user's bucket name from database
    */
-  async getUploadPresignedUrl(objectKey: string): Promise<string> {
-    return await this.getMinioService().getUploadPresignedUrl(this.bucketName, objectKey);
+  private async getUserBucketName(userId: number): Promise<string> {
+    const { services } = await this.getServices();
+    const user = await services.user.findOne({ id: userId });
+    if (!user || !user.bucketName) {
+      throw new Error('User not found or bucket name not set');
+    }
+    return user.bucketName;
```

Comment:
calling api /user/me will return all information about user so we dont need this APIAPI